CMakeLists.txt

```diff
@@ -13,7 +13,9 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-cmake_minimum_required (VERSION 3.9)
+cmake_minimum_required (VERSION 3.12)
```

Comment:
Will it be ok for the OpenROAD project that we increase the CMake required version?

Reply:
What is this change is driving the need to upgrade cmake?

Reply:
Cmake 3.9 should be able to support CUDA libraries well. I changed it because the lowest version I have used is 3.12 not 3.9. I will change it back.